src/Menu/menuman.h

```diff
@@ -146,7 +146,7 @@ typedef struct DATA_INFO
 {
   char                        field_0[2];
   char                        field_2;
-  char                        field_3;      // block count
+  char                        blocksCount;
```

Comment:
One last nitpick, could you use snake case for these names?

Reply:
No problem, but I can't do it before tomorrow.